Assets/MRTK/Providers/Oculus/XRSDK/Editor/OculusXRSDKConfigurationChecker.cs

```diff
@@ -49,72 +44,16 @@ static OculusXRSDKConfigurationChecker()
         /// - Save the Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef file
         /// This will result in Unity reloading the assembly with the appropriate dependencies.
         /// </remarks>
-        private static void UpdateAsmDef()
+        private static void UpdateScriptingDefinitions()
         {
-            FileInfo[] asmDefFiles = FileUtilities.FindFilesInAssets(AsmDefFileName);
-
-            if (asmDefFiles.Length == 0)
-            {
-                Debug.LogWarning($"Unable to locate file: {AsmDefFileName}");
-                return;
-            }
-            if (asmDefFiles.Length > 1)
-            {
-                Debug.LogWarning($"Multiple ({asmDefFiles.Length}) {AsmDefFileName} instances found. Modifying only the first.");
-            }
-
-            AssemblyDefinition asmDef = AssemblyDefinition.Load(asmDefFiles[0].FullName);
-            if (asmDef == null)
-            {
-                Debug.LogWarning($"Unable to load file: {AsmDefFileName}");
-                return;
-            }
-
-            List<string> references = new List<string>();
-            if (asmDef.References != null)
-            {
-                references.AddRange(asmDef.References);
-            }
-
-            bool changed = false;
-
 #if UNITY_2019_3_OR_NEWER
-            List<VersionDefine> versionDefines = new List<VersionDefine>();
-            if (asmDef.VersionDefines != null)
-            {
-                versionDefines.AddRange(asmDef.VersionDefines);
-            }
 
-            if (!references.Contains(OculusReference))
-            {
-                // Add a reference to the XR SDK Oculus assembly
-                references.Add(OculusReference);
-                changed = true; 
-            }
-
-            if (!versionDefines.Contains(OculusDefine))
-            {
-                // Add the Oculus #define
-                versionDefines.Add(OculusDefine);
-                changed = true;
-            }
+            ScriptUtilities.AppendScriptingDefinitions(BuildTargetGroup.Android, Definitions);
```

Comment:
Is this needed? This looks like new behavior that this script didn't used to do, and this `#define` is already in the asmdef:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/233ed7d00b8850f7982ce77eabd5b4732ffdaab3/Assets/MRTK/Providers/Oculus/XRSDK/Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef#L25-L31